homeassistant/components/homeassistant_sky_connect/__init__.py

```diff
@@ -24,19 +39,44 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         # The USB dongle is not plugged in
         raise ConfigEntryNotReady
 
-    usb_info = usb.UsbServiceInfo(
-        device=entry.data["device"],
-        vid=entry.data["vid"],
-        pid=entry.data["pid"],
-        serial_number=entry.data["serial_number"],
-        manufacturer=entry.data["manufacturer"],
-        description=entry.data["description"],
-    )
+    addon_manager: AddonManager = get_addon_manager(hass)
+    try:
+        addon_info: AddonInfo = await addon_manager.async_get_addon_info()
+    except AddonError as err:
+        _LOGGER.error(err)
+        raise ConfigEntryNotReady from err
+
+    # Start the addon if it's not started
```

Comment:
We only need to start the add-on if the user is using multi-pan. If it’s not, it should do nothing. I don’t see that case handled right now?

Reply:
It's handled, this case starts the addon if it's installed but not started. If the addon is not installed, we won't start it.

Reply:
Where do we check the SkyConnect config entry data ?

Reply:
d532a6a